lib/defaults/defaults.go

```diff
@@ -762,23 +751,33 @@ var (
 
 	// FIPSCiphers is a list of supported FIPS compliant SSH ciphers.
 	FIPSCiphers = []string{
-		"aes128-ctr",
-		"aes192-ctr",
+		"aes256-gcm@openssh.com",
 		"aes256-ctr",
+		"aes192-ctr",
 		"aes128-gcm@openssh.com",
+		"aes128-ctr",
 	}
 
 	// FIPSKEXAlgorithms is a list of supported FIPS compliant SSH kex algorithms.
 	FIPSKEXAlgorithms = []string{
 		"ecdh-sha2-nistp256",
 		"ecdh-sha2-nistp384",
-		"echd-sha2-nistp521",
```

Comment:
This was a typo and never worked. This was explicitly removed by BoringSSL and by Go. See https://go-review.googlesource.com/c/go/+/549975.